pyuploadtool/releases_hosting_provider/github_releases.py

```diff
@@ -55,9 +55,9 @@ def create_release(self, metadata: ReleaseMetadata, artifacts):
             self.logger.warning("push to default branch, creating continuous release")
 
             # not using "latest", as this value is reserved by GitHub
-            metadata.tag = "continuous"
-            metadata.release_name = "Continuous build"
-            prerelease = True
+            metadata.tag = os.getenv("GITHUB_CONTINUOUS_RELEASE_TAG", "continuous")
+            metadata.release_name = os.getenv("GITHUB_CONTINUOUS_RELEASE_NAME", "Continuous build")
+            prerelease = True if os.getenv("GITHUB_CONTINUOUS_RELEASE_IS_STABLE") else False
```

Comment:
I am not really sure if this is the right way to do it. This would fail, if the environment variable is defined like:
```bash
GITHUB_CONTINUOUS_RELEASE_IS_STABLE=false ./pyuploadtool*.AppImage ...
```

Reply:
You will have to set up a small repo to test any of this. This is my repo for this purpose: https://github.com/TheAssassin/uploadtool-actions-test/

I don't like the variable name, by the way. How about `GITHUB_CONTINUOUS_RELEASE_TYPE`?

Reply:
> I don't like the variable name, by the way. How about GITHUB_CONTINUOUS_RELEASE_TYPE?

Ah, that would solve the problem, and looks better.

Reply:
Hence I suggested to avoid using a "boolean" value. You do not want to parse that. If you insist on a "flag" (as I'd call it, similar to CLI flags like `--abc`), you should call it `GITHUB_CONTINUOUS_STABLE_RELEASE`. That way, you can just document that whenever it is set to *any* value, you are going to take the branch. You just need to `if os.getenv("..."):` (will also exclude empty values, some people don't know how to `unset` but rather `export VAR=""`).

But I still would strongly recommend using a `...RELEASE_TYPE` kind of variable, where people specify a value of `stable` if needed. You can precisely check the allowed options, e.g., `stable` and `prerelease`, and reject all unknown values. If the value is not provided or an empty string, you can log that and set the value to a defined default, e.g., `prerelease`.

Reply:
>  and reject all unknown values. 

Should we raise an error in that case?

Reply:
That would be my recommendation. I personally don't like any kind of implicit behavior. Sure, your build may fail if you make a typo. However, as a user, you won't be surprised by any kind of implicit behavior. That's my philosophy when designing user interfaces, and a CLI is a user interface, too.